src/backend/metal/src/command.rs

```diff
@@ -123,7 +123,8 @@ struct State {
     index_buffer: Option<IndexBuffer>,
     rasterizer_state: Option<native::RasterizerState>,
     pipeline_depth_stencil: Option<(pso::DepthStencilDesc, metal::DepthStencilState)>,
-    dynamic_depth_stencil: Option<metal::DepthStencilDescriptor>,
+    dynamic_depth_stencil_desc: Option<metal::DepthStencilDescriptor>,
```

Comment:
I tried collapsing these into one field but it was surprisingly difficult to make it work with `set_stencil_mask_values`, maybe somebody else has an idea?